accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,502 @@
+
+### Non-Goals
+
+* Replace TFMs or expand runtime identifiers (RIDs)
+* Support multi-targeting between OS versions
+
+## Design
+
+| TFM             | Compatible With                                            | Comments                          |
+|-----------------|------------------------------------------------------------|-----------------------------------|
+| net5.0          | net1..4 (with NU1701 warning)                              | No WinForms or WPF                |
+|                 | netcoreapp1..3.1 (warning when WinForms/WPF is referenced) |                                   |
+|                 | netstandard1..2.1                                          |                                   |
+| net5.0-android  | xamarin.android                                            |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-ios      | xamarin.ios                                                |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-macos    | xamarin.macos                                              |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-tvos     | xamarin.tvos                                               |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-watchos  | xamarin.watchos                                            |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-win      | netcoreapp1..3.1                                           | WinForms + WPF and probably WinUI |
+|                 | (+everything else inherited from net5)                     |                                   |
+| Tizen, Unity... | Will follow the Xamarin model                              |                                   |
+
+***Open Issue**. In case of Xamarin, there is a discussion of whether the
+convergence with .NET Core will be a breaking change for existing NuGet packages
+(for example, there is a desire to change their namespaces). In that case, we
+should not map it to the existing TFM as that would be pointless.*
+
+***Open Issue**. Will the above scheme work for iPad?*
+
+### Mapping to properties
+
+There are three existing MSBuild properties:
+
+| Property                        | Meaning            | Examples                         |
+|---------------------------------|--------------------|----------------------------------|
+| TargetFramework (TFM)           | The short name     | `net4`, `netcoreapp3.0`          |
```

Comment:
```suggestion
| TargetFrameworkMoniker (TFM)           | The short name     | `net4`, `netcoreapp3.0`          |
```

Reply:
That change is not correct, its called TargetFramework.  Please call this "friendly name" and not "short name". It is not a canonical representation, unlike the combination of TFI, TFV, TFP.

Reply:
I mean, indeed the property is `TargetFramework`, but TFM stands for `Target Framework Moniker` as it can be seen on this doc : [Target frameworks in SDK-style projects](https://docs.microsoft.com/en-us/dotnet/standard/frameworks#latest-target-framework-versions)

![image](https://user-images.githubusercontent.com/17796723/74593472-d3799d80-502b-11ea-9cc3-be49259de6d2.png)

Reply:
@joeltankam this column is about MSBuild properties, not names in the UI.

Reply:
> Please call this "friendly name" and not "short name". It is not a canonical representation, unlike the combination of TFI, TFV, TFP.

Good point, fixed/